src/components/contact.tsx

```diff
@@ -357,8 +335,11 @@ export function Contact({ isOpen, onClose, dark }) {
           </StyledBoxContentWrapper>
         </StyledBox>
 
-        <StyledContactWrapper>
-          <StyledSquare dark={dark}> Contact </StyledSquare>
+        <StyledContactWrapper onClick={stopPropagation}>
+          <StyledSquare dark={dark} onClick={stopPropagation}>
```

Comment:
Have a look at `navbar.tsx`, the dark prop is being passed incorrectly, resulting in this being the wrong colour